buildpipeline/portable-linux.groovy

```diff
@@ -58,6 +58,7 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:rhel7_prereqs_2') {
             def targetHelixQueues = ['Centos.73.Amd64.Open',
                                      'RedHat.73.Amd64.Open',
                                      'Debian.87.Amd64.Open',
+                                     'Debian.90.Amd64.Open',
```

Comment:
You'd basically need to create a job that just ran this item, and make it optional using the trigger.  There's no special knowledge of what is going on here, that array is just concatenated together to make a command line.

So basically, you could parameterize the pipeline so that it only ran Debian 9 for instance, then set up the trigger so that it ran that job when you commented the correct thing.